Libraries/LibWeb/CSS/FontFace.cpp

```diff
@@ -25,6 +25,8 @@
 #include <LibWeb/WebIDL/Buffers.h>
 #include <LibWeb/WebIDL/Promise.h>
 
+#include <harfbuzz/hb.h>
```

Comment:
This is pretty minor and I suppose won't really matter since we won't be swapping out harfbuzz any time soon, but considering harfbuzz is only used internally in LibGfx/Font at the moment without appearing in any public API, does it make sense to have this in LibGfx instead?